lib/jazzy/sourcekitten.rb

```diff
@@ -361,7 +361,12 @@ def self.make_source_declarations(docs, parent = nil)
         declaration.parent_in_code = parent
         declaration.type = SourceDeclaration::Type.new(doc['key.kind'])
         declaration.typename = doc['key.typename']
-        current_mark = SourceMark.new(doc['key.name']) if declaration.type.mark?
+        documented_name = if Config.instance.hide_declarations == 'objc'
+                            doc['key.swift_name']
+                          else
+                            doc['key.name']
+                          end
```

Comment:
Sometimes there is no Swift decl for an ObjC decl, ie. cursorInfo returns nothing.  Not sure how well code will react to having nil for `documented_name` hereon down.

Would be nice to factor the config queries into config to cut down on the number of literal strings scattered around!

Reply:
Done.